webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -6231,4 +6231,79 @@ body:not(.accesibility) :focus {
 }
 body.accesibility :focus {
 	outline: 5px auto #3b99fc;
-}
\ No newline at end of file
+}
+
+/* Chat Popup Template Code Starts */
+
+.chat-popup-widget-container {
+    box-shadow: 0 2px 9px 2px rgba(0, 0, 0, 0.17);
+    background-color: #ffffff;
+    position: absolute;
+	min-height: 70px;
```

Comment:
@CaptainT33mo  please fix code indentation

Reply:
@akshatdhabi It is already correct. Looking weird in Github.